src/web/src/pages/_app.tsx

```diff
@@ -13,6 +13,8 @@ import '../styles/globals.css';
 import '@fontsource/spartan';
 import '@fontsource/pt-serif';
 
+import 'react-lite-youtube-embed/dist/LiteYouTubeEmbed.css';
```

Comment:
Should this be in the post component where we use it vs. here?  If we don't load that component, it's not needed.

Reply:
Yeah, it should be in the `Post` component. I saw an example leaving it in the `_app.tsx`, so I didn't know it could be in another place.